### PR TITLE
pAI cleanup and tweaks

### PR DESCRIPTION
**Pull request by jknpj**
A few changes to make future changes easier:

* Made pAIs use their own DNA var to store their master's DNA instead of the old master_dna var
* Made the Food Synth unhardcoded
* Made the Chem Synth medical chems unhardcoded

Also added albuterol to their portable pharmacy because it's the kind of niche non-powergaming chem that they usually handle.

:cl:
 * rscadd: pAIs with both Medical Supplement and Chem Synth apps now have access to Albuterol

### CODE REVIEW

File: code/game/objects/items/devices/paicard.dm [hunk 1]
```diff
@@ -79,35 +73,32 @@
...
 	if(href_list["setdna"])
-		if(pai.master_dna)
+		if(pai.dna)
 			return
-		var/mob/M = usr
-		if(!istype(M, /mob/living/carbon))
-			to_chat(usr, "<font color=blue>You don't have any DNA, or your DNA is incompatible with this device.</font>")
+		var/mob/NewMaster = usr
+		if(!iscarbon(NewMaster))
+			to_chat(NewMaster, "<font color=blue>You don't have any DNA, or your DNA is incompatible with this device.</font>")
```
Comment (jknpj) [inline code/game/objects/items/devices/paicard.dm:84]: >blue

**Comment by utilitas-** [comment]
_Add methylin._

**Comment by brndd** [comment]
nanui pais when

**Comment by jknpj** [comment]
Whenst thou codeth it

**Comment by ShiftyRail** [comment]
pAI software datums when

**Comment by jknpj** [comment]
> Whenst thou codeth it

**Pull request merged**